src/libs/Navigation/Navigation.js

```diff
@@ -262,6 +263,31 @@ function setIsNavigationReady() {
     resolveNavigationIsReadyPromise();
 }
 
+/**
+ * @param {{address: Object, legalFirstName: String, legalLastName: String, phoneNumber: String}} privatePersonalDetails
+ * @param {Array<String>} loginList
+ */
+function goToNextPhysicalCardRoute(privatePersonalDetails, loginList) {
+    const {address, legalFirstName, legalLastName, phoneNumber} = privatePersonalDetails;
+    const currentRoute = navigationRef.current && navigationRef.current.getCurrentRoute();
+    const {domain} = (currentRoute && currentRoute.params) || {domain: ''};
```

Comment:
If there can be a fallback to an empty string for `domain` should we handle it somehow? 

Seems like it would be unexpected if that happened based on the fact that we use it to navigate below.